src/main/java/frc/robot/Robot.java

```diff
@@ -86,6 +105,11 @@ public void autonomousPeriodic() {
    */
   @Override
   public void teleopPeriodic() {
+    driveSpeed();
+  }
+
+  public void driveSpeed() {
```

Comment:
This method should at least have a Javadoc one-line description, as the others do.

Reply:
A one-line description has been added to this function.